src/pudl/workspace/datastore.py

```diff
@@ -267,26 +286,18 @@ def get_descriptor(self, dataset: str) -> DatapackageDescriptor:
                 )
         return self._descriptor_cache[doi]
 
-    def get_resource_key(self, dataset: str, name: str) -> PudlResourceKey:
-        """Returns PudlResourceKey for given resource."""
-        return PudlResourceKey(dataset, self._dataset_to_doi[dataset], name)
-
-    def get_doi(self, dataset: str) -> str:
-        """Returns DOI for given dataset."""
-        return self._dataset_to_doi[dataset]
+    def get_resource_key(self: Self, dataset: str, name: str) -> PudlResourceKey:
```

Comment:
This isn't used anywhere but the unit test - should we make this a private method?

Reply:
Sure, that makes sense.

Reply:
DELETED!